manifests/config/passenger.pp

```diff
@@ -59,6 +59,8 @@
 #
 # $ipa_authentication::       Whether to install support for IPA authentication
 #
+# $apache_start_timeout::       Startup time for Apache
```

Comment:
The alignment is off. It'd also mention this is only for systemd and replace time with timeout.